src/main/kotlin/no/nav/familie/klage/brev/BrevRepository.kt

```diff
@@ -1,10 +1,20 @@
 package no.nav.familie.klage.brev
 
 import no.nav.familie.klage.brev.domain.Brev
+import no.nav.familie.klage.brev.domain.BrevmottakereJournalposter
 import no.nav.familie.klage.repository.InsertUpdateRepository
 import no.nav.familie.klage.repository.RepositoryInterface
+import org.springframework.data.jdbc.repository.query.Modifying
+import org.springframework.data.jdbc.repository.query.Query
 import org.springframework.stereotype.Repository
 import java.util.UUID
 
 @Repository
-interface BrevRepository : RepositoryInterface<Brev, UUID>, InsertUpdateRepository<Brev>
+interface BrevRepository : RepositoryInterface<Brev, UUID>, InsertUpdateRepository<Brev> {
+
+
+    @Modifying
+    @Query("""UPDATE brev SET mottakere_journalposter = :brevmottakereJournalposter WHERE behandling_id = :behandlingId""")
+    fun oppdaterMottakerJournalpost(behandlingId: UUID, brevmottakereJournalposter: BrevmottakereJournalposter)
```

Comment:
Har valgt å kun oppdatere mottakere her i stedet for å oppdatere hele brevet. Ulempen er at sporbar ikke oppdateres.. Hva tenker dere?